src/server/graphql/models/User/userMutation.js

```diff
@@ -151,7 +151,13 @@ export default {
         createdAt: new Date(userInfo.created_at),
         tms: userInfo.tms
       };
-      const {email, id: userId, picture, preferredName} = auth0User;
+      const {email, id: userId, createdAt, picture, preferredName} = auth0User;
```

Comment:
this should be after the `if (currentUser)` conditional. if you care about preferredName & avatar being up to date, we should also include this in the `updateUserProfile` mutation. A user is only guaranteed to call this mutation once. After that, they could just be getting fresh tokens via the connection handler.